Another build issue

Building for 3.36
```
In file included from /usr/include/evolution/e-util/e-util.h:93,
                 from /usr/include/evolution/shell/e-shell-view.h:26,
                 from /home/grosse/checkout/evolution-taskwarrior/src/m-mail-ui.c:25:
/usr/include/evolution/e-util/e-contact-store.h:30:10: fatal error: libebook/libebook.h: No such file or directory
   30 | #include <libebook/libebook.h>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [src/CMakeFiles/taskwarrior-module.dir/build.make:89: src/CMakeFiles/taskwarrior-module.dir/m-mail-ui.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:150: src/CMakeFiles/taskwarrior-module.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

--- a/README.md
+++ b/README.md
@@ -16,7 +16,7 @@ There might be some unneeded code being left around.
  * Requires evolution >= 3.36.0
 
 **Note**: If evolution is installed on ubuntu using `apt`, you probably need to install the
-development header package `evolution-dev` and `libebook1.2-dev`, as well.
+development header packages `evolution-dev` and `libebook1.2-dev`, as well.
 
 ## Installation
 For evolution >= 3.40.0:

--- a/README.md
+++ b/README.md
@@ -16,7 +16,7 @@ There might be some unneeded code being left around.
  * Requires evolution >= 3.36.0
 
 **Note**: If evolution is installed on ubuntu using `apt`, you probably need to install the
-development header package `evolution-dev`, as well.
+development header package `evolution-dev` and `libebook1.2-dev`, as well.
 
 ## Installation
 For evolution >= 3.40.0:

